app/src/main/java/ru/myuniquenickname/myapplication/presentation/movieList/MovieListFragment.kt

```diff
@@ -118,12 +125,61 @@ class MovieListFragment : Fragment() {
     }
 
```

Comment:
The function calls each time when the Fragment creates. And makes searching with the latest value stored in LiveData.